### PR TITLE
Serial tidy

### PR DESCRIPTION
**Pull request by jamesgorrie**
* Update labels
* Add background to serial page
* Add spacing to series navigation
* Only use 1 serial

### CODE REVIEW

**Comment by jamesgorrie** [comment]
screen shot 2018-10-02 at 09 42 01

* adds background 
* removes link from non existent content

File: common/styles/utilities/_root-scope-classes.scss [hunk 1]
```diff
@@ -423,6 +423,22 @@
   width: auto;
 }
 
+.image-ls-overlay {
```
Comment (davidpmccormick) [inline common/styles/utilities/_root-scope-classes.scss]: what does `ls` mean?

Comment (davidpmccormick) [reply common/styles/utilities/_root-scope-classes.scss]: …and where are these classes being used?

Comment (jamesgorrie) [reply common/styles/utilities/_root-scope-classes.scss]: It was `L`-s, as in the plural of `L`.
It's not and it's been removed.

**Pull request merged**